sdk/include/xdk/ketypes.h

```diff
@@ -120,6 +124,54 @@ typedef struct _SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX {
   } DUMMYUNIONNAME;
 } SYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX, *PSYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX;
 
+typedef enum _CPU_SET_INFORMATION_TYPE
+{
+    CpuSetInformation
+} CPU_SET_INFORMATION_TYPE, *PCPU_SET_INFORMATION_TYPE;
+
+//_Struct_size_bytes_(Size)
```

Comment:
This is present in the newer SDK? Since it appears we support this annotation (other typedefs have it), why did you choose to keep it commented there?